microsoft-dynamics-365/2.4/modules/ROOT/pages/microsoft-dynamics-365-connector-studio.adoc

```diff
@@ -0,0 +1,130 @@
+= Use Anypoint Studio to Configure Microsoft Dynamics 365 Connector 2.4  - Mule 4
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector-studio.adoc
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector-design-center.adoc
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector
+. Configure the connector
+. Configure an input source for the connector
+
+== Add the Connector to Your Project
+
```

Comment:
This is the old template. We should make a list of connectors that need to be updated to match the current template.